Testshell/test_shell.cpp

```diff
@@ -62,27 +62,50 @@ class TestShell {
   }
 
   void executeCommand(const Command &command) {
-    if (command.command == "read") {
+    string cmd = command.command;
+    if (cmd == "read") {
       read(command);
-    } else if (command.command == "write") {
+    } else if (cmd == "write") {
 
-    } else if (command.command == "fullread") {
+    } else if (cmd == "fullread") {
 
-    } else if (command.command == "fullwrite") {
+    } else if (cmd == "fullwrite") {
 
-    } else if (command.command == "help") {
-      help();
-    } else if (command.command == "1_") {
+    } else if (cmd == "help") {
 
-    } else if (command.command == "2_") {
+    } else if (cmd == "1_" || cmd == "1_FullWriteAndReadCompare") {
+      bool isFailed = executeScriptOne(command);
+      if (isFailed) return;
+    } else if (cmd == "2_") {
 
-    } else if (command.command == "3_") {
+    } else if (cmd == "3_") {
 
     } else {
       cout << "INVALID COMMAND" << endl;
     }
   }
 
+  bool executeScriptOne(const Command &command) {
+    string valueStr = command.args[0];
+    unsigned long value = std::stoul(valueStr, nullptr, 16);
+
+    for (int lba = 0; lba < 100; lba += 4) {
+      for (int i = 0; i < 4; i++) {
+        ssd->write(lba + i, value);
+      }
+
+      for (int i = 0; i < 4; i++) {
+        string result = ssd->getResult();
+        if (result != valueStr) {
```

Comment:
요구사항이 좀 모호하게 되어 있는 데, 실제로 read 한 후에 읽어와야 하지 않을까요?

> 0 ~ 4번 LBA까지 실제 저장된 값과 맞는지 확인

```suggestion
      for (int i = 0; i < 4; i++) {
        ssd->read(lba);
        string result = ssd->getResult();
        if (result != valueStr) {
```

Reply:
아 read 호출이 필요하겠군요 수정하겠습니다!